app-crypt/swtpm/swtpm-0.1.0.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{4,5,6,7} )
+
+inherit autotools python-any-r1
+
+DESCRIPTION="Libtpms-based TPM emulator"
+HOMEPAGE="https://github.com/stefanberger/swtpm"
+SRC_URI="https://github.com/stefanberger/swtpm/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="gnutls fuse test"
+
+COMMON_DEPEND="dev-libs/libtpms
+	dev-libs/openssl:0
+	fuse? ( sys-fs/fuse:0
+		 >=dev-libs/glib-2.0.0 )
+	gnutls? ( >=net-libs/gnutls-3.1.0[tools]
+		  dev-libs/libtasn1 )"
```

Comment:
While at it, please sort the deps too alphabetically :)

In lexicographic ordering `><` dont weigh before alphabet order, 
```
§ export LC_COLLATE=C
$ echo -e "sys-fs/fuse:0\n>=dev-libs/glib-2.0.0" | sort
>=dev-libs/glib-2.0.0
sys-fs/fuse:0
```
```
	fuse? (
		>=dev-libs/glib-2.0.0
		sys-fs/fuse:0
	)
	gnutls? (
		dev-libs/libtasn1
		>=net-libs/gnutls-3.1.0[tools]
	)"
```

glib is slotted, so you should add a slot operator there, instead of using `>=`.
libtasn has a subslot operator, so you should bind it using `:=` if it links during compilation, like it seems it does since its put in DEPEND.

Reply:
Both fixed